lofty.com.website.json

```diff
@@ -82,16 +75,17 @@
       "txtConflictMatchingMode": "All"
     },
     {
-      "type": "SPFM",
+      "type": "TXT",
       "groupId": "email",
-      "host": "@",
+      "host": "mail",
       "ttl": 3600,
-      "spfRules": "include:amazonses.com"
+      "data": "v=spf1 include:amazonses.com ~all",
+      "txtConflictMatchingMode": "All"
```

Comment:
This would prevent any TXT records on this host. For SPF actually prefix would be the right approach.